spec/features/course/assessment/submission/submissions_spec.rb

```diff
@@ -34,6 +35,12 @@
         create(:submission, :graded, assessment: assessment, course: course,
                                      creator: course_staff.user)
       end
+      let!(:staff_attempt_submission) do
+        create(:submission, :attempting, assessment: mcq_assessment, course: course,
+                                         creator: course_staff.user)
+      end
+      let(:options) { mcq_assessment.questions.first.specific.options }
```

Comment:
Line 42 and 43 are not necessary to be abstracted.

Reply:
I think this abstraction is so that the line 55 won't be long and easier to understand?

Reply:
You can create a variable within the context?

Reply:
done